src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -410,6 +410,8 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                             source: $source,
                             image: $runtime['image'],
                             version: $version,
+                            cpus: 2,
+                            memory: 1024,
```

Comment:
why is this hard coded here?

Reply:
We decided on using the functions limits right? reverting